lib/src/showcase_widget.dart

```diff
@@ -83,6 +85,7 @@ class ShowCaseWidgetState extends State<ShowCaseWidget> {
   late Duration autoPlayDelay;
   late bool autoPlayLockEnable;
   late bool enableAutoScroll;
+  late bool disableOverlayClick;
```

Comment:
Just noticed that all these late properties are being initialized from the widget in `initState` but are not updated in `didUpdateWidget`. That means it won't be updated if the values are changed in the widget.

Possible fixes:
1. Override `didUpdateWidget` and update them there!
2. If they only exist so that that data can be accessed from outside, then all of them can just be getters that delegate to the same property from the widget. This fix is much better than above fix.

Reply:
@MehulKK It would be great if you could refactor this as well!